samples/contrib/versioned-pipeline-ci-samples/kaggle-ci-sample/download_dataset/download_data.py

```diff
@@ -0,0 +1,31 @@
+"""
+step #1: download data from kaggle website, and push it to gs bucket
+"""
+
+def processAndUpload(
+    bucket_name
+):
+    from google.cloud import storage
+    storage_client = storage.Client()
+    bucket = storage_client.get_bucket(bucket_name)
+    train_blob = bucket.blob('train.csv')
+    test_blob = bucket.blob('test.csv')
+    train_blob.upload_from_filename('train.csv')
+    test_blob.upload_from_filename('test.csv')
+
+    with open('train.txt', 'w') as f:
+        f.write('gs://'+bucket_name+'/train.csv')
```

Comment:
We probably want those datas sets in a public available place instead of asking users to prepare the data sets. E.g., gs://ml-pipeline-playground/shakespeare1.txt in pipeline "[Sample] Basic - Parallel execution".

Reply:
I think this sample maybe a little different from other samples. If put the data in a public gs bucket, user may not get the full process of completing a kaggle competition, from download,  train, to submit.